sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -1485,11 +1485,12 @@ class RocksDB(
       override def log(infoLogLevel: InfoLogLevel, logMsg: String) = {
         // Map DB log level to log4j levels
         // Warn is mapped to info because RocksDB warn is too verbose
+        // Info is mapped to debug because RocksDB info is too verbose
```

Comment:
Some RocksDB info logs have actually been useful for us for debugging. Which specific logs did we see most often here ?

Reply:
This part is the biggest pain for us and I've been trying to find ways to filter these out. All the statistics are incredibly verbose, and when you are running with multiple cores per executor and they all just get intermingled it's basically impossible to find anything useful. +1 to this being a thing to opt-in to debug logs for debugging purposes

Reply:
Particularly because it appears to spit out the statistics at some regular interval for every opened state store instance on the executor, which can accumulate to a lot and then drown out the logs of what the executor is actually doing

Reply:
@siying - is there some way to disable the periodic stats reporting in RocksDB ?

Reply:
@anishshri-db ,
>Which specific logs did we see most often here ?

Mainly `Hard Linking <filename>.log`
eg: 25/03/19 10:38:55 INFO RocksDB [StateStoreId(opId=0,partId=265,name=default), queryRunId=30d232ab-0d2a-454d-a217-5b2debfe10e9, queryId=4c8ca990-9fd4-4b69-8f90-ce933b3f4ef3]: [NativeRocksDB-1] [ilities/checkpoint/checkpoint_impl.cc:127] Hard Linking 009723.log